mailersend/base/base.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+import requests
+
+API_BASE = "https://api.mailersend.com/v1"
+API_KEY = os.environ.get("MAILERSEND_API_KEY")
```

Comment:
not used?

Reply:
is it also possible to offload this to the client, this gives the developer the best of both worlds, if they want to use an ENV they can pass it in 

see boto as a good example of how AWS do credentials